### PR TITLE
Careers page

### PR DESCRIPTION
**Pull request by ajithranka**
Resolves or continues [AR-1269](https://team-1624093970686.atlassian.net/browse/AR-1269).

## Changes

- Adds a careers page
- Adds a new variant for `v-tabs`

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: components/careers/EndSection.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <section>
+    <v-stack
+      direction="column"
+      md-direction="column"
+      sm-direction="column"
```
Comment (shrinathprabhu) [inline components/careers/EndSection.vue:6]: I guess a single direction is fine unless direction is changed for specific resolution. Can you try and check if that works?

Comment (ajithranka) [reply components/careers/EndSection.vue]: Good catch 👌🏼. Just `direction="column"` works for all breakpoints.

I've updated the code.

File: components/careers/HeroSection.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<template>
+  <section>
+    <v-container>
+      <v-stack
+        direction="column"
+        md-direction="column"
+        sm-direction="column"
```
Comment (shrinathprabhu) [inline components/careers/HeroSection.vue:7]: I guess we can use just `direction` prop if the value is not changing in any resolution. Can you try it and check if it works? If it works, we won't require `md-direction` and `sm-direction`. Not sure if it's required for `md-gap` and `sm-gap`

**Review by shrinathprabhu — CHANGES_REQUESTED** [review]
Rest all looks good, just added one comment

**Review by shrinathprabhu — APPROVED** [review]

**Pull request merged**